vehicle/mkvi/BUILD

```diff
@@ -0,0 +1,11 @@
+load("//bazel/tools:defs.bzl", "dbc_gen")
+
+# Generates DBC
+dbc_gen(
+    name = "mkvi.dbc",
+    srcs = [
+        "//libs/can/test:test.yml",
+        "//vehicle/mkvi/software/emeter:emeter.dbc",
+        "//vehicle/mkvi/software/motor_controller:motor_controller.dbc",
```

Comment:
We should just copy these files into a common folder, like `vehicle/common` and update the mkv DBC as well.